packages/react/src/components/Card/Card.jsx

```diff
@@ -516,6 +516,7 @@ const Card = (props) => {
                 triggerClassName={`${iotPrefix}--card--header--tooltip`}
                 id={`card-tooltip-${id}`} // https://github.com/carbon-design-system/carbon/pull/6744
                 triggerText=""
+                iconDescription={`card-header-tooltip-${id}`} // To fix accessibility violation.
```

Comment:
While we are doing this we may want to add the ability to provide a translatable string from the user. So we check if there is an `i18n.iconDescription`. If so we use that and if not we can use this as a fallback.